underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java

```diff
@@ -70,8 +69,8 @@ public boolean supportsPath(String path) {
    *
    * @return true if both access and secret key are present, false otherwise
    */
-  private boolean addAndCheckGoogleCredentials() {
-    return Configuration.containsKey(PropertyKey.GCS_ACCESS_KEY)
-        && Configuration.containsKey(PropertyKey.GCS_SECRET_KEY);
+  private boolean addAndCheckGoogleCredentials(UnderFileSystemConfiguration conf) {
```

Comment:
it seems #3740 failed to rename it correctly when I am implementing the `Configuration` with `PropertyKey` enums. Fixed